actix-web/src/test/mod.rs

```diff
@@ -10,12 +10,17 @@
 //! # Calling Test Service
 //! - [`TestRequest`]
 //! - [`call_service`]
+//! - [`try_call_service`]
 //! - [`call_and_read_body`]
+//! - [`try_call_and_read_body`]
 //! - [`call_and_read_body_json`]
+//! - [`try_call_and_read_body_json`]
 //!
 //! # Reading Response Payloads
 //! - [`read_body`]
+//! - [`try_read_body`]
 //! - [`read_body_json`]
+//! - [`try_read_body_json`]
```

Comment:
Looks like I didn't re-export these with `pub use` below according to doc lint. 

- [ ] re-export new test helpers in mod.rs